tests/common.go

```diff
@@ -4042,6 +4043,25 @@ func GetAzureCredsFromEnv() (tenantID, clientID, clientSecret, subscriptionID, a
 	return tenantID, clientID, clientSecret, subscriptionID, accountName, accountKey
 }
 
+// GetNfsCredsFromEnv get creds for nfs share.
+func GetNfsCredsFromEnv() (nfsServerAddress, nfsPath, nfsSubPath, nfsMountOptions string) {
+	nfsServerAddress = os.Getenv("NFS_SERVER_ADDR")
+	expect(nfsServerAddress).NotTo(equal(""),
+		"NFS_SERVER_ADDR Environment variable should not be empty")
+
+	nfsPath = os.Getenv("NFS_PATH")
+	expect(nfsPath).NotTo(equal(""),
+		"NFS_PATH Environment variable should not be empty")
+
+	nfsSubPath = os.Getenv("NFS_SUB_PATH")
+	expect(nfsSubPath).NotTo(equal(""),
```

Comment:
NFS sub path can be empty, right?

Reply:
For Automation usecase, its mandatory to have subpath.